pages/common/helix.md

```diff
@@ -1,12 +1,12 @@
-# vim
+# helix
 
 > Helix, A post-modern text editor, provides several modes for different kinds of text manipulation.
 > Pressing `i` enters insert mode. `<Esc>` enters normal mode, which enables the use of Helix commands.
 > More information: <https://helix-editor.com/>.
 
 - Open a file:
 
-`helix {{path/to/file}}`
+`hx {{path/to/file}}`
```

Comment:
Wait, both are correct. You should create an alias page for hx -> helix.
```suggestion
`helix {{path/to/file}}`
```

Reply:
`helix` is not valid on my machine (MacOS); this is what prompted me to change it in the first place. Which platform are you on? And I guess we should have per-platform versions since it doesn't work on Mac.

Reply:
I checked the Arch Linux package and that package provides both commands, long and short.